index.html

```diff
@@ -26,7 +26,12 @@ <h1>Welcome to the student showcase!</h1>
         <div>SIX</div>
         <div>SEVEN</div>
         <div>EIGHT</div>
-        <div>NINE</div>
+        <div> 
+            <h3>Fabiane Clubb</h3> 
+           <img src="https://media.licdn.com/dms/image/C4E03AQGeXeHQcJNl0w/profile-displayphoto-shrink_200_200/0?e=1546473600&v=beta&t=l1YTiid7CyOnu_54qqqm-U9ZcL_yWSC8fIEw0sJzAZg"/> 
+            <h1>"The more you weigh, the harder to kidnap. Eat Cake".</h1>
+            <p><i class="fas fa-envelope-square"></i> email<a href="melofabi@ymail.com">email</p>
```

Comment:
Remember that font awesome only works if you've added the library through a `<link>` tag. You are free to do that, though 😉